jdaviz/configs/mosviz/plugins/slit_overlay/slit_overlay.py

```diff
@@ -97,10 +114,11 @@ def place_slit_overlay(self, *args, **kwargs):
                 x_coords = pix_rec.vertices.x
                 y_coords = pix_rec.vertices.y
 
-                fig_image = self.app.get_viewer("image-viewer").figure
+                image_viewer = self.app.get_viewer(self._default_image_viewer_reference_name)
+                fig_image = image_viewer.figure
 
-                if self.app.get_viewer("image-viewer").toolbar.active_tool is not None:
-                    self.app.get_viewer("image-viewer").toolbar.active_tool = None
+                if image_viewer.toolbar.active_tool is not None:
+                    image_viewer.toolbar.active_tool = None
```

Comment:
out of scope since it isn't introduced here, but I'm curious why enabling the slit overlay disables any selected tool 🤔